src/Authenticator/LoginHandler.php

```diff
@@ -585,15 +585,15 @@ public function getMember()
     }
 
     /**
-     * @return SessionStore
+     * @return StoreInterface
      */
-    protected function getSessionStore()
+    protected function getStore()
     {
-        if (!$this->sessionStore) {
-            $this->sessionStore = SessionStore::create($this->getRequest());
+        if (!$this->store) {
+            $this->store = Injector::inst()->create(StoreInterface::class, $this->getRequest());
```

Comment:
I think this means that we should add the constructor to the interface as we pass in the request here as a constructor arg.